lib/web_console/templates/console.js.erb

```diff
@@ -490,13 +567,15 @@ REPLConsole.prototype.renderInput = function() {
   }
 
   this.promptDisplay.appendChild(document.createTextNode(before));
-  promptCursor.appendChild(document.createTextNode(current));
-  this.promptDisplay.appendChild(promptCursor);
+  this.promptDisplay.appendChild(center);
   this.promptDisplay.appendChild(document.createTextNode(after));
+
+  var hint = this.autocomplete && !this.autocomplete.confirmed && this.autocomplete.getSelectedWord();
```

Comment:
@gsamokovarov Thanks for catching it! I wanted to break an infinite loop (`finish` => `setInput` => `refine` => `finish` => ....)  by the second condition. Now it is fixed by 9547e5f6.